agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/exporter/AgentLogExporter.java

```diff
@@ -111,7 +111,17 @@ public CompletableResultCode export(Collection<LogRecordData> logs) {
           itemCount = Math.round(100.0 / samplingPercentage);
         }
 
-        TelemetryItem telemetryItem = mapper.map(log, stack, itemCount);
+        String type =
+            log.getAttributes()
+                .get(
+                    com.azure.monitor.opentelemetry.exporter.implementation.SemanticAttributes
+                        .EXCEPTION_TYPE);
+        String message =
+            log.getAttributes()
+                .get(
+                    com.azure.monitor.opentelemetry.exporter.implementation.SemanticAttributes
+                        .EXCEPTION_MESSAGE);
+        TelemetryItem telemetryItem = mapper.map(log, stack, type, message, itemCount);
```

Comment:
(and move the code inside the method)
```suggestion
        TelemetryItem telemetryItem = mapper.map(log, stack, itemCount);
```